cmake/onnxruntime_python.cmake

```diff
@@ -511,6 +512,15 @@ if (onnxruntime_USE_NNAPI_BUILTIN)
         $<TARGET_FILE_DIR:${build_output_target}>/onnxruntime/capi/
   )
 endif()
+
+if (onnxruntime_USE_COREML_BUILTIN)
```

Comment:
onnxruntime_USE_COREML 

The reason NNAPI is `onnxruntime_USE_COREML_BUILTIN` is that we used to have 2 NNAPI EPs (the other one was removed)